audio/softsynth/eas.cpp

```diff
@@ -280,27 +270,10 @@ int MidiDriver_EAS::open() {
 	// number of buffer fills per readBuffer()
 	_rounds = INTERMEDIATE_BUFFER_SIZE / (_config->bufSize * _config->channels);
 
-	debug("EAS initialized (voices:%d channels:%d rate:%d buffer:%d) "
+	debug("EAS initialized (voices:%ld channels:%ld rate:%ld buffer:%ld) "
 			"tempo:%u rounds:%u", _config->voices, _config->channels,
 			_config->rate, _config->bufSize, _baseTempo, _rounds);
 
-	// TODO doesn't seem to work with midi streams?
-	if (ConfMan.hasKey("soundfont")) {
-		const Common::String dls = ConfMan.get("soundfont");
-
-		debug("loading DLS file '%s'", dls.c_str());
-
-		EASFile f;
-		memset(&f, 0, sizeof(EASFile));
-		f.path = dls.c_str();
-
-		res = _loadDLSFunc(_EASHandle, 0, &f);
-		if (res)
-			warning("error loading DLS file '%s': %d", dls.c_str(), res);
-		else
-			debug("DLS file loaded");
-	}
```

Comment:
So, why has this been removed? And is there any other way of loading sound fonts?

Reply:
I already explained why in the relevant commit message. As far as I know, there isn't any other way of loading soundfonts with EAS.

Reply:
Perhaps we could use the Android version for this? This change was probably in the codebase in Android version 5.1 and later versions:
https://en.m.wikipedia.org/wiki/Android_version_history

Reply:
Maybe a configure check is more robust than just relying on Android version?